src/main/java/previewcode/backend/api/v1/PullRequestAPI.java

```diff
@@ -52,6 +54,33 @@ public PrNumber createPR(@PathParam("owner") String owner,
         return number;
     }
 
+
+    /**
+     * Updates the ordering
+     *
+     * @param owner
+     *            The owner of the repository where the pull request was made
+     * @param name
+     *            The owner of the repository where the pull request was made
+     * @param number
+     *            The number of the pull request
+     * @param body
+     *            The new ordering of the pull request
+     * @return The number of the newly made pull request
+     */
+    @POST
+    @Path("{number}/ordering")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public void updateOrdering(@PathParam("owner") String owner,
+                             @PathParam("name") String name,
+                             @PathParam("number") String number, List<Ordering> body){
```

Comment:
No, as this is an incorrect type.

![image](https://cloud.githubusercontent.com/assets/5946438/25120744/95a71662-241f-11e7-8af1-d659fef5d4fd.png)